src/core/FilePatternDescriptor.ts

```diff
@@ -0,0 +1,7 @@
+
+interface FilePatternDescriptor {
+  pattern: string;
+  included: boolean;
+}
+
+export default FilePatternDescriptor;
```

Comment:
@simondel It would be easy to also add a `mutated` property to the file pattern. Would it be confusing for the `mutate` array? We could also remove the `mutate` array entirely, but than people are forced to use a config file rather than command line.

What do you think?